Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: improving UI during connection flow

### Description (*)

Adding loading icon to popup panel while starting host and making URelay health check call and disabling Confirm button during health check call.
Adding check to only call OnOnlineModeDropdownChanged() when the value changes.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-1480](https://jira.unity3d.com/browse/MTT-1480)
### Manual testing scenarios

1. Launch game with high artificial latency (helps see this fix better)
2. Open Start Host popup panel
3. Select Unity Relay as connection method
4. See the loading animation appearing and the Confirm button disabling while waiting for the health check call
5. Start hosting a game using Unity Relay or Photon Relay
6. See the Loading popup before the game is properly started and the character selection scene is joined
### Questions or comments

### Contribution checklist
